### DISCUSSION TITLE
Anything similar to the Google Coral for AMD?

### DISCUSSION THREAD

**Discussion by catch-twenty-two** [ROCm/ROCm#3950] [Q&A] [answered]
Hello,

I'd like to take a look at the ROCm code base and start playing with it as an alternative to CUDA. I don't have enough room for another GPU in my setup. (Currently have an NVIDIA GPU) Is there a device similar to the Google Coral that will fit in my USB port and provide acceleration compatibility with the ROCm code base?

**Comment by darren-amd** [comment, marked as answer]
Hi @catch-twenty-two,

Glad to hear that you're interested in trying ROCm. I'm not aware of anything similar to the Google Coral that supports ROCm, however, one suggestion would be to try using an eGPU enclosure with an AMD GPU (you can find the latest supported hardware [here](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html)). 

Additionally, we do actually support Nvidia hardware for HIP, but I'm not sure which part of the ROCm stack you're interested in: [HIP on Nvidia](https://rocm.docs.amd.com/projects/HIP/en/latest/install/install.html#installation).

**Comment by jamesxu2** [comment]
@catch-twenty-two You may also be interested in the discussion on this ticket: https://github.com/ROCm/ROCm/issues/1917

**Comment by catch-twenty-two** [comment]
Great, thanks to both of you for the pointers!
